api/app/Http/Controllers/AuthController.php

```diff
@@ -54,16 +56,17 @@ public function login(Request $request)
             throw new RuntimeException($e->getMessage(), 500, $e);
         }
 
-        return $this->item(new AuthToken(['token' => $token]));
+        return response($transformer->transformItem(new AuthToken(['token' => $token])));
```

Comment:
I'm not a fan of the global function `response()` (it just doesn't feel like a good pattern), should we move it to the base transformer so we can call `$this->response()` in all controllers?